test/playback.test.js

```diff
@@ -59,6 +59,36 @@ QUnit.test('Advanced Bip Bop', function(assert) {
   });
 });
 
+QUnit.test('replay', function(assert) {
+  let done = assert.async();
+
+  assert.expect(2);
+  let player = this.player;
+
+  player.autoplay(true);
```

Comment:
do you have to autoplay? Seems like this would be annoying to manage with autoplay restrictions

Reply:
This is to simplify the tests and not have to wait for loadstart before playing. Autoplay restrictions aren't a problem because the player is muted

Reply:
I could remove the autoplay setting from all the tests and just create the player with autoplay on